flink-cdc-common/src/test/java/org/apache/flink/cdc/common/types/utils/DataTypeUtilsTest.java

```diff
@@ -0,0 +1,132 @@
+import java.util.stream.IntStream;
+
+import static org.apache.flink.table.api.DataTypes.ARRAY;
+import static org.apache.flink.table.api.DataTypes.BIGINT;
+import static org.apache.flink.table.api.DataTypes.BINARY;
+import static org.apache.flink.table.api.DataTypes.BOOLEAN;
+import static org.apache.flink.table.api.DataTypes.BYTES;
+import static org.apache.flink.table.api.DataTypes.CHAR;
+import static org.apache.flink.table.api.DataTypes.DATE;
+import static org.apache.flink.table.api.DataTypes.DECIMAL;
+import static org.apache.flink.table.api.DataTypes.DOUBLE;
+import static org.apache.flink.table.api.DataTypes.FIELD;
+import static org.apache.flink.table.api.DataTypes.FLOAT;
+import static org.apache.flink.table.api.DataTypes.INT;
+import static org.apache.flink.table.api.DataTypes.MAP;
+import static org.apache.flink.table.api.DataTypes.ROW;
+import static org.apache.flink.table.api.DataTypes.SMALLINT;
+import static org.apache.flink.table.api.DataTypes.STRING;
+import static org.apache.flink.table.api.DataTypes.TIME;
+import static org.apache.flink.table.api.DataTypes.TIMESTAMP_LTZ;
+import static org.apache.flink.table.api.DataTypes.TIMESTAMP_WITH_TIME_ZONE;
+import static org.apache.flink.table.api.DataTypes.TINYINT;
+import static org.apache.flink.table.api.DataTypes.VARBINARY;
+import static org.apache.flink.table.api.DataTypes.VARCHAR;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/** A test for the {@link org.apache.flink.cdc.common.types.utils.DataTypeUtils}. */
+public class DataTypeUtilsTest {
+    private static final DataType[] allTypes =
+            new DataType[] {
+                DataTypes.BOOLEAN(),
+                DataTypes.BYTES(),
+                DataTypes.BINARY(10),
+                DataTypes.VARBINARY(10),
+                DataTypes.CHAR(10),
+                DataTypes.VARCHAR(10),
+                DataTypes.STRING(),
+                DataTypes.INT(),
+                DataTypes.TINYINT(),
+                DataTypes.SMALLINT(),
+                DataTypes.BIGINT(),
+                DataTypes.DOUBLE(),
+                DataTypes.FLOAT(),
+                DataTypes.DECIMAL(6, 3),
+                DataTypes.DATE(),
+                DataTypes.TIME(),
+                DataTypes.TIME(6),
+                DataTypes.TIMESTAMP(),
+                DataTypes.TIMESTAMP(6),
+                DataTypes.TIMESTAMP_LTZ(),
+                DataTypes.TIMESTAMP_LTZ(6),
+                DataTypes.TIMESTAMP_TZ(),
+                DataTypes.TIMESTAMP_TZ(6),
+                DataTypes.ARRAY(DataTypes.BIGINT()),
+                DataTypes.MAP(DataTypes.SMALLINT(), DataTypes.STRING()),
+                DataTypes.ROW(
+                        DataTypes.FIELD("f1", DataTypes.STRING()),
+                        DataTypes.FIELD("f2", DataTypes.STRING(), "desc")),
+                DataTypes.ROW(DataTypes.SMALLINT(), DataTypes.STRING())
+            };
+
+    @Test
+    public void testToFlinkDataType() {
```

Comment:
```suggestion
    void testToFlinkDataType() {
```

Reply:
I get the error `Method 'testToFlinkDataType' annotated with '@Test' should be public `

Reply:
In JUnit5, test  class and methods do not need to be declared as public.

Reply:
Get it, Done.